packages/n/nextpnr/package.yml

```diff
@@ -0,0 +1,29 @@
+name       : nextpnr
+version    : 0.6
+release    : 1
+source     :
+    - https://github.com/YosysHQ/nextpnr/archive/refs/tags/nextpnr-0.6.tar.gz : 76fa4bca48cc8462a8a3c28f89673439e7632a9baabc7bb121c58b75936d2d0b
+homepage   : https://github.com/YosysHQ/nextpnr
+license    : ISC
+component  : office.scientific
+summary    : A portable FPGA place and route tool.
+description: |
+    nextpnr is a vendor-neutral, timing-driven, FOSS FPGA placement and routing tool.
+patterns   :
+    - ice40 : /usr/bin/*-ice40
+    - generic : /usr/bin/*-generic
+builddeps  :
+    - pkgconfig(Qt5Core)
+    - pkgconfig(eigen3)
+    - pkgconfig(python3)
+    - icestorm
+    - libboost-devel
+    - yosys
+setup      : |
+    %cmake \
```

Comment:
Use `%cmake_ninja` unless for whatever reason this doesn't work with the ninja build generator.